examples/targets.py

```diff
@@ -1,7 +1,7 @@
 import logging
+
 import ppb
-from ppb import Vector
-from ppb import keycodes
+from ppb import keycodes, Vector
```

Comment:
FYI, there were already places which had multiple imports per line, and in `ppb-vector` too.

Reply:
Yeah, she and I kinda disagree on that.

Reply:
OK, so then @pathunstrom, can we agree to disagree, instead of expecting contributors to conform with a styleguide that only exists in your head, that the codebase doesn't comply with, and giving them imperative-mode feedback without explanation why?

Reply:
PS: I just realised the tone might be wrong there; to clarify, I wasn't annoyed at you, though I felt it was important to provide feedback on that being frustrating & why (not necessarily just for me, but also for other contributors), esp. since that's not the first time it happens.